core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -58,9 +55,19 @@
  */
 public final class CoreSocketFactory {
 
-  public static final String CLOUD_SQL_INSTANCE_PROPERTY = "cloudSqlInstance";
-  public static final String CLOUD_SQL_DELEGATES_PROPERTY = "cloudSqlDelegates";
-  public static final String CLOUD_SQL_TARGET_PRINCIPAL_PROPERTY = "cloudSqlTargetPrincipal";
+  @Deprecated
```

Comment:
Could we add a comment on what people should use instead here (ConnectionConfig)?